docs/docsite/rst/roadmap/COLLECTIONS_8.rst

```diff
@@ -0,0 +1,57 @@
+.. _ansible_8_roadmap:
+
+===================
+Ansible project 8.0
+===================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-core <https://pypi.org/project/ansible-core/>`_ package as well. All dates are subject to change. See the `ansible-core 2.15 Roadmap <https://docs.ansible.com/ansible-core/devel/roadmap/ROADMAP_2_15.html>`_ for the most recent updates on ``ansible-core``.
+
+.. contents::
+   :local:
+
+
+Release schedule
+=================
+
+
+:2023-03-31: ansible-core feature freeze, stable-2.15 branch created.
+:2023-04-03: Start of ansible-core 2.15 betas (biweekly, as needed).
+:2023-04-04: Ansible-8.0.0 alpha1 (roughly biweekly ``ansible`` alphas timed to coincide with ``ansible-core-2.15`` pre-releases).
+:2023-04-19: Community topic: List any backwards incompatible collection releases that beta1 should try to accommodate.
```

Comment:
```suggestion
```
I think we can remove this one, we haven't really done this for the last two releases anyway.